applications/templates/applications/sign_in.html

```diff
@@ -1,12 +1,50 @@
 {% extends "base.html" %}
 
-{% load snippet %}
+{% block metatitle %}Apply: Sign In | OpenSAFELY Jobs{% endblock metatitle %}
+
+{% block breadcrumbs %}
+<nav class="breadcrumb-container" aria-label="breadcrumb">
+  <div class="container">
+    <ol class="breadcrumb rounded-0 mb-0 px-0">
+      <li class="breadcrumb-item">
+        <a href="/">Home</a>
+      </li>
+      <li class="breadcrumb-item">
+        <a href="{% url 'applications:start' %}">Apply</a>
+      </li>
+      <li class="breadcrumb-item active" aria-current="page">
+        Sign In
+      </li>
+    </ol>
+  </div>
+</nav>
+{% endblock breadcrumbs %}
+
+{% block jumbotron %}
+<div class="jumbotron jumbotron-fluid pt-md-2">
+  <div class="container">
+    <div class="d-flex flex-column flex-md-row align-items-center">
+      <div>
+        <h1 class="display-4">Sign in to GitHub</h1>
+
+        <p class="lead">
+          If your application is approved, you will be able to submit study
+          code to run on OpenSAFELY via GitHub.
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+{% endblock jumbotron %}
 
 {% block content %}
 <div class="container">
   <div class="row">
     <div class="col">
-      {% snippet "sign_in" %}
```

Comment:
What's the motivation behind moving the text here?

Reply:
There was so little text left in the snippet that it didn't seem worth the downsides of breaking locality of behaviour in favour of a sentence or two.  I have a feeling this page is unlikely to change too much as well so really just making a call!